api/backend/ecs/load_balancer_manager.go

```diff
@@ -301,26 +317,53 @@ func (this *ECSLoadBalancerManager) createLoadBalancer(
 		scheme = "internet-facing"
 	}
 
-	subnets, _, err := this.getSubnetsAndAvailZones(isPublic)
+	subnets, _, err := e.getSubnetsAndAvailZones(isPublic)
 	if err != nil {
 		return err
 	}
 
-	if _, err := this.ELB.CreateLoadBalancer(ecsLoadBalancerID.String(), scheme, securityGroups, subnets, listeners); err != nil {
+	if _, err := e.ELB.CreateLoadBalancer(ecsLoadBalancerID.String(), scheme, securityGroups, subnets, listeners); err != nil {
 		return err
 	}
 
 	return nil
 }
 
-func (this *ECSLoadBalancerManager) UpdateLoadBalancer(loadBalancerID string, ports []models.Port) (*models.LoadBalancer, error) {
-	model, err := this.GetLoadBalancer(loadBalancerID)
+func (e *ECSLoadBalancerManager) UpdateLoadBalancerHealthCheck(loadBalancerID string, healthCheck models.HealthCheck) (*models.LoadBalancer, error) {
+	ecsLoadBalancerID := id.L0LoadBalancerID(loadBalancerID).ECSLoadBalancerID()
+	if err := e.updateHealthCheck(ecsLoadBalancerID, healthCheck); err != nil {
+		return nil, err
+	}
+
+	model, err := e.GetLoadBalancer(loadBalancerID)
```

Comment:
[nit] This block is redundant since the healthcheck is already populated in `populateModel`. Should be:
```
func (e *ECSLoadBalancerManager) UpdateLoadBalancerHealthCheck(loadBalancerID string, healthCheck models.HealthCheck) (*models.LoadBalancer, error) {
    ecsLoadBalancerID := id.L0LoadBalancerID(loadBalancerID).ECSLoadBalancerID()
    if err := e.updateHealthCheck(ecsLoadBalancerID, healthCheck); err != nil {
        return nil, err
    }

    return e.GetLoadBalancer(loadBalancerID)
}
```